package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.1.3",
   "description": "A namespaced stylish logger primarily for the browser",
   "main": "index.js",
```

Comment:
SSG supports the browser field, which replaces main with browser value when in browser